src/frontend/index.tsx

```diff
@@ -44,6 +44,14 @@ export let intl: IntlShape;
 // Wait for the DOM to load before we scour it for an element that requires React to render
 document.addEventListener('DOMContentLoaded', async event => {
   try {
+    // Plyr uses CustomEvents which we consume in our xapi event infrastructure
+    try {
+      // We need to specifically attempt to use the constructor as `CustomEvent` exists in some browsers
+      // (eg. IE11) but does not support the constructor.
+      const test = new CustomEvent('CustomEventConstructorIsSupported');
```

Comment:
you don't need this assignement, just creating the instance should be enough, you will don't use the instance after.

Reply:
The linter complains when I don’t assign the instance. Figured instead of adding a linter comment I might as well do this

Reply:
F**king linter :smile: keep it like this.